src/instruments/src/EFB/Efb.tsx

```diff
@@ -118,9 +119,23 @@ const navigraph = new NavigraphClient();
 const Efb = () => {
     const history = useHistory();
 
+    const [currentLocalTime] = useSimVar('E:LOCAL TIME', 'seconds', 3000);
+    const [, setBrightness] = useSimVar('L:A32NX_EFB_BRIGHTNESS', 'number');
+    const [usingAutobrightness] = useSimVar('L:A32NX_EFB_USING_AUTOBRIGHTNESS', 'bool', 5000);
+
+    // handle setting brightness if user is using autobrightness
+    useEffect(() => {
+        if (usingAutobrightness) {
+            const localTime = currentLocalTime / 3600;
+            // eslint-disable-next-line no-restricted-properties
+            setBrightness(Math.pow(Math.E, -(Math.pow(localTime - 12, 2) / 24)) * 100);
```

Comment:
interesting how we use three different approaches across the plane to automatically adjust the display brightness.

Reply:
For all I know, other solutions are based on breakpoints of either day/night or a couple more in between. This approach uses a bell curve function which I think would much better match the functionality of a sensor that detects ambient light intensity.